distribution/pom.xml

```diff
@@ -70,6 +70,26 @@
             <groupId>org.hibernate</groupId>
             <artifactId>hibernate-search-backend-jgroups</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-search-jsr352-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>javax.inject</groupId>
+            <artifactId>javax.inject</artifactId>
+        </dependency>
```

Comment:
Why did you add those dependencies to javax.inject and javax.batch? Is this necessary to generate the javadoc?

Reply:
Yup. There are references to them in the analyzed code we need them. That's why we already have most of the other dependencies.